packages/app-root/next.config.js

```diff
@@ -0,0 +1,4 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {}
+
+module.exports = nextConfig
```

Comment:
`module.exports` is CommonJS. Should we switch to `import` and `export`? They’ve been supported by Node since version 12 (the other two Next.js apps were originally written for Node 8.)

Reply:
We should also run the new app on Node 20.

Reply:
Like the next.conifg.mjs example here? https://nextjs.org/docs/app/api-reference/next-config-js

Reply:
Yes, but more generally do we want to be using `require` or `import` in our Node scripts now? All the rest of our code is using `import` and `export`, but we've stuck with `require` and `module.exports` for anything that runs exclusively in Node.

Reply:
True! I'll change the next config file to .mjs for now. I'm still thinking about a good way to document FEM's progress toward ESM-only, and the consequences/benefits of that path forward. Maybe a Github Discussion branching from the already existing Issue documenting mocha's incompatibility with ESM-only.